### PR TITLE
Add more operators

### LINKED ISSUE #101: Add Gt, Gte, Lt, Lte operators
Same behaviour as https://github.com/heetch/regula/blob/b79b8edceafa6f3c5d7461ba29f83cbbb94ddb40/rule/eq_expr.go#L8-L14

### PR DESCRIPTION
**Pull request by YassineE**
This PR adds the remaining operators of #103.

It fixes #101, #104, #105, #106.

### CODE REVIEW

File: rule/expr.go [hunk 1]
```diff
@@ -461,6 +581,111 @@ func (v *Value) GT(other *Value) bool {
 	return false
 }
 
+// GTE reports whether v is greater or equal than other.
+func (v *Value) GTE(other *Value) bool {
+	switch v.Type {
+	case "bool":
+		v1, _ := strconv.ParseBool(v.Data)
```
Comment (tealeg) [inline rule/expr.go]: :tulip: We're doing a whole load of this  `strconv.Parse...` stuff, in the 0.7 version we moved this to common functions.  I don't think it's worth doing a lot of re-factoring here, though, so I stuck a :tulip: on it ;-)

Comment (YassineE) [reply rule/expr.go]: Yeah I opted for the Quick&Dirty here because I know that this will be removed in the 0.7.

**Comment by YassineE** [comment]
@asdine 🆙

**Comment by asdine** [comment]
Waiting for this discussion to be done, I have the same issue in all the PRs https://github.com/heetch/regula/pull/103#discussion_r271624571

**Comment by YassineE** [comment]
@asdine @tealeg 🆙 🙏

**Pull request merged**